app-games/src/main/res/values/strings.xml

```diff
@@ -143,6 +143,10 @@
   <string name="contact_us_button">Contact Us</string>
   <string name="try_again_or_contact_us_body">Please try again or contact us.</string>
   <string name="done_button">Done!</string>
+  <string name="iap_appc_title">Pay with the AppCoins Wallet</string>
```

Comment:
Don't add strings. hardcode the strings in code with a TODO comment like "//TODO hardcoded string" so it can be changed in future